docs/concepts/packaging-types.adoc

```diff
@@ -53,5 +52,23 @@ executing the hollow JAR:
 ----
 $ java -jar myapp-hollow-swarm.jar myapp.war
 ----
-endif::[]
+
+=== Pre-Build hollow jars
+
+Wildfly Swarm ships three pre-build hollow jar types, for different use cases:
+
+* full: The main Java EE related capabilities
+* web: A smaller functional scope, focused on web technologies
+* microprofile: The capabilities belonging the microprofile specification
```

Comment:
"belonging to the MicroProfile specifications": missing "to", wrong capitalization of "MicroProfile", plural "specifications"

Also I guess the descriptions are somewhat vague intentionally, but I think we can do a little better. Something like:

- `full`: all the Java EE capabilities
- `web`: limited set of Java EE capabilities, focused on web-related technologies
- `microprofile`: same as the `web` hollow JAR plus the Eclipse MicroProfile capabilities